### ISSUE TITLE
Underscore in tag name/value

### ISSUE THREAD

**Issue by mmindenhall** [peh/node-i2cssh#4]
Hi again!

In your readme you say "underscore separated as it is forbidden to be used in tags".  Perhaps that used to be the case, but underscores are currently allowed in both tag names and values (I just tested this on an instance).

In our EC2, *all* of our `Name` tags contain underscores (e.g., we have autoscaling groups where each instance has something like `Name=vpc_stage_whatever`).  Passing `-t Name_vpc_stage_whatever` currently fails due to splitting on underscore (so the value is split into 4 strings instead of 2).

My first thought was to suggest using a different character as the separator, but the ones that came to mind (`:` and `|`) are also valid in tag names/values (I checked).

Hence, I think it would work best to have an option in .i2csshrc to override the separator character.  For example:

```yaml
---
aws:
  region: us-east-1               # the aws region to doi calls to
  separator: ":"                  # separator between tag & value in '-t'
  usePrivateDns: false            # whether to use private dns names
  only:                           # only check the following tag key's
    - Name
clusters:
  stageapp:
    hosts:
      - stage1.example.com
      - stage2.example.com
```

With that config, this command should succeed:

```
$ envchain aws i2cssh -u ubuntu -t Name:vpc_stage_whatever
```

In addition, it might be worth using a different character as the default separator value (one less likely to require users to override the default).  

If this isn't something you have time for, I could probably take a crack at a PR.  Thanks!

**Comment by peh** [comment]
fixed with #5

**Comment by peh** [comment]
released 1.6.1
